src/rebar_digraph.erl

```diff
@@ -99,7 +101,7 @@ cull_deps(Graph, Vertices, Level, Levels, Solution, Discarded) ->
                                                      DiscardedAcc1}
                                             end
                                     end, {NewVertices, SolutionAcc, LevelsAcc, DiscardedAcc}, OutNeighbors)
-                    end, {[], Solution, Levels, Discarded}, lists:keysort(1, Vertices)),
+                    end, {[], Solution, Levels, Discarded}, lists:sort(Vertices)),
```

Comment:
I'm curious about what changed there? Should it work better with `rebar_utils:tup_sort/1` ?